test/testconfig.py

```diff
@@ -108,3 +108,6 @@ def create_transaction(db):
     OCI_NAMESPACE_WHITELIST: List[str] = []
 
     FEATURE_USER_INITIALIZE = True
+
+    FEATURE_QUOTA_MANAGEMENT = True
+    DEFAULT_SYSTEM_REJECT_QUOTA_BYTES = 0
```

Comment:
`None` instead of `0`. `None` makes it clearer that we don't want that restriction, whereas `0` could mean either no pushes at all, or no limits at all.

Reply:
This was the requested behavior, We can re-evaluate after this PR.